numba/targets/arraymath.py

```diff
@@ -2520,6 +2520,15 @@ def np_corrcoef_impl_single_variable(x, y=None, rowvar=True):
 #----------------------------------------------------------------------------
 # Element-wise computations
 
+@overload(np.argwhere)
+def np_argwhere(a):
+
+    def impl(a):
+        return np.transpose(np.vstack(np.nonzero(a)))
```

Comment:
I think `a` could have `np.asarray()` called on it first, tuples, scalars etc ought to just work.
```pycon
In [15]: np.argwhere((1, 2, 3))                                                                                                  
Out[15]: 
array([[0],
       [1],
       [2]])

In [16]: np.argwhere(1)                                                                                                         
Out[16]: array([[0]])

```